src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+
+  const fromDateObject = {};
+  const toDateArray = [];
+
+  let index = 0;
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (dateArray[index] < 30) {
+        fromDateObject['YYYY'] = '20' + dateArray[index];
+      } else {
+        fromDateObject['YYYY'] = '19' + dateArray[index];
+      }
```

Comment:
Move year converting logic to separate function

Reply:
done